src/main/java/se/omegapoint/conferre/proposal/domain/supportive/Conference.java

```diff
@@ -0,0 +1,11 @@
+package se.omegapoint.conferre.proposal.domain.supportive;
+
+import se.omegapoint.conferre.Entity;
+import se.omegapoint.conferre.conference.event.ConferenceCreated;
+
+public class Conference extends Entity {
```

Comment:
Inte helt övertygad här. Skapar lite bakvänt beroende i en entitet att ta in eventdata i konstruktorn tycker jag. Kanske bryta ut dataurtaget ur eventet (bara conferenceId i detta fall) till någon slags disassembler/builder?